app/assets/javascripts/services/topology_service.js

```diff
@@ -65,5 +65,29 @@ ManageIQ.angular.app.service('topologyService', function() {
     return { x: 0, y: 9, r: 17 };
   };
 
+  this.getItemStatusClass = function(d) {
+    switch (d.item.status) {
+      case "OK":
+      case "On":
+      case "Ready":
+      case "Running":
+      case "Succeeded":
+      case "Valid":
+        return "Success";
```

Comment:
Can we make the class names lowercase?

And looks like this PR may be missing the actual class definitions, since you're no longer setting the stroke color directly..?